packages/network-controller/src/NetworkController.ts

```diff
@@ -333,6 +337,13 @@ type CustomNetworkClientId = string;
  */
 type NetworkClientId = BuiltInNetworkClientId | CustomNetworkClientId;
 
+/**
+ * Data related to a particular network.
```

Comment:
Considering this is just a plural version of `NetworkDetails` (or whatever the name ends up being), should we make the two more similar? So judging by the above this would be:

``` suggestion
 * Information about networks not held by any other part of state.
```

(although feel free to adjust this description in both places if it's not accurate or is confusing)